projects/packages/search/src/widgets/class-search-widget.php

```diff
@@ -1053,4 +1054,32 @@ class="widefat filter-count"
 		</div>
 		<?php
 	}
+
+	/**
+	 * Returns the taxonomies supported for users to choose from.
+	 */
+	protected function get_supported_taxonomies() {
+		return array_filter(
+			get_taxonomies( array( 'public' => true ), 'objects' ),
+			function ( $taxonomy ) {
+				return in_array(
+					$taxonomy->name,
+					/**
+					 * Filters the taxonomies that shows in filter drop downs of the search widget.
+					 *
+					 * @since  $$next-version$$
+					 *
+					 * @param array $taxonomies_to_show List of taxonomies that shown for search widget.
+					 */
+					apply_filters(
+						'jetpack_search_supported_taxonomies',
```

Comment:
I feel like this filter could be confusing to users because they will assume that adding a taxonomy makes it work, but it won't.